packages/contracts-bedrock/src/cannon/libraries/MIPSSyscalls.sol

```diff
@@ -99,7 +99,7 @@ library MIPSSyscalls {
 
     uint32 internal constant SCHED_QUANTUM = 100_000;
     /// @notice Start of the data segment.
```

Comment:
Will this impact the bytecode of the `MIPS` contract?

Reply:
Do we need this to be part of the `MIPS` abi? cc @mbaxter  @Inphi @clabby ?

Reply:
I don't think so

Reply:
This doesn't change the bytecode, based on the nil version snapshot change.